#This combines to define our path for the topics
# system: "telebot" #This is running on the telebot
# level: "L_1" #The level of the system
# subsystem: "motion" #The system name, generally should be the package name

dummy: 
  ros__parameters:
    list_param: [0,1,-1,-1000]
    str_param: "mystr"
    int_param: 12

# multiplexer:
#   ros__parameters:
#     control_sources:
#       minibot_control: ["topic1,topic2"]
#       gui_control: ["eg1,eg2"]

# #Parameters for upper driver
driver_upper:
  ros__parameters:
    pro_motors: [4, 5, 11, 12, 30, 31]
    xm_motors: [1, 2, 3, 6, 7, 8, 9, 10, 13, 14, 15, 16, 17, 21, 22, 23, 24].
    usb_device: "/dev/ttyUSB0"
    usb_baudrate: 1000000

